pyigm/cgm/utils.py

```diff
@@ -16,22 +16,27 @@
 
 from pyigm.field.galaxy import Galaxy
 
-def calc_rho(galaxy, igm_sys, cosmo, ang_sep=None, correct_lowz=True):
-    """
+
+def calc_rho(galaxy, igm_sys, cosmo, ang_sep=None, correct_lowz=True, Galactic=False):
+    """ Calculate the impact parameter between the galaxy and IGM sightline
+
     Parameters
     ----------
     galaxy : Galaxy object
     igm_sys : IGMSystem object or list
     cosmo : astropy.cosmology
+    Galactic : bool, optional
+      Calculate for our Galaxy!
```

Comment:
We should add guidance in the docstring about ang_sep and correct_lowz!  I think that is my bad, actually.